### DISCUSSION TITLE
Optional custom Context menu maybe?

### DISCUSSION THREAD

**Discussion by Blatman** [neutralinojs/neutralinojs#630] [Ideas]
Just thought that a custom context menu feature (right-click on window)might be worthwhile. At the moment a right-click on the window will show a standard context menu with Reload and Inspect Element options (on Mac at least) and while useful in a dev scenario the standard options may not be the best in an end user scenario eg: Inspect Element.
Also when the transparency/opacity feature for windows is implemented then widget type apps become possible (frameless/rounded corners) and context menus can be useful in this mode - it could be an option to have a custom menu on a per window basis. It may be the case that some apps don't want the right-click menu to show at all in production mode.

**Comment by ratatoeskr666** [comment]
I'm ready to implement this feature. Here is my ongoing pull request for part 1 (disabling/enabling the context menu): https://github.com/neutralinojs/neutralinojs/pull/1263
A second PR will likely follow with the ability to fully customize the context menu.
